lib/node_modules/@stdlib/repl/base/ctor/lib/drain.js

```diff
@@ -178,9 +156,7 @@ function drain( repl ) {
 
 		if ( error ) {
 			debug( 'Error: %s', error.message );
-			repl._ostream.write( 'Error: '+error.message+'\n' );
-			repl.emit( 'command', code.raw, false ); // command failed
-			displayPrompt( repl, false );
+			repl.emit( 'command', code.raw, false, error, false ); // command failed
```

Comment:
What are the additional `error` and `false` arguments here? The `onCommand` listener, TMK, only has `cmd` and `success` arguments.